content/en/docs/components/pipelines/concepts/pipeline-root.md

```diff
@@ -1,7 +1,7 @@
 +++
 title = "Pipeline Root"
 description = "Getting started with Kubeflow Pipelines pipeline root"
-weight = 50
+weight = 15
 
 +++
 {{% beta-status
```

Comment:
We can remove this `{{% beta-status ...` flag, because this is a core part of KFP v2.

Reply:
Done.